src/NCRings.jl

```diff
@@ -155,6 +155,13 @@ julia> is_unit(ZZ(-1)), is_unit(ZZ(4))
 """
 function is_unit end
 
+# Default implementation: covers all integral domains
+function is_nilpotent(a::T) where { T <: NCRingElem }
+  is_domain_type(typeof(a)) && return is_zero(a)
```

Comment:
```suggestion
function is_nilpotent(a::T) where {T <: NCRingElem}
  is_domain_type(T) && return is_zero(a)
```
Julia convention is to write `{T <: NCRingElem}` not `{ T <: NCRingElem }`

Reply:
I can do what you prefer, but there are **lots** of other functions which use different spacing in the `where` clause _e.g._ no curly brackets, no spaces around `<:`,...